game-core/src/test/java/games/strategy/triplea/delegate/RevisedTest.java

```diff
@@ -780,8 +782,9 @@ public void testAaOwnership() {
     MustFightBattle battle =
         (MustFightBattle) AbstractMoveDelegate.getBattleTracker(gameData).getPendingBattle(fic, false, null);
     // fight
-    ScriptedRandomSource randomSource = new ScriptedRandomSource(0, 5);
-    delegateBridge.setRandomSource(randomSource);
+    whenGetRandom(delegateBridge)
+        .thenReturn(new int[] {0})
+        .thenReturn(new int[] {5});
```

Comment:
Is there a specific reason to chain the thenReturn calls like this, or just a design choice over the variable arguments variant of thenReturn?

Reply:
This issue is moot after 2c4e4dea7.  There is no `thenAnswer()` overload that takes a varargs parameter.